webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  Flex,
+  FlexItem,
+  GridItem,
+  Label,
+} from '@patternfly/react-core';
+
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import { translate as __ } from 'foremanReact/common/I18n';
+import ContentViewIcon from '../../../../scenes/ContentViews/components/ContentViewIcon';
+
+
+const ContentViewDetailsCard = ({ hostDetails }) => {
+  let contentViewDetails;
+  if (hostDetails.content_facet_attributes) {
+    const cv = hostDetails.content_facet_attributes.content_view;
+    const env = hostDetails.content_facet_attributes.lifecycle_environment;
+    const versionId = hostDetails.content_facet_attributes.content_view_version_id;
+    const version = hostDetails.content_facet_attributes.content_view_version;
+
+    contentViewDetails = (
+      <DescriptionList isAutoColumnWidths>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('Content View')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <Flex>
+              <FlexItem spacer={{ default: 'spacerNone' }}><ContentViewIcon composite={cv.composite} /></FlexItem>
+              <FlexItem><a href={`/content_views/${cv.id}`}>{`${cv.name}`}</a> </FlexItem>
+              <FlexItem><Label color="purple" href={`/lifecycle_environments/${env.id}`}>{`${env.name}`}</Label></FlexItem>
+            </Flex>
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('Version in use')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <Flex>
+              <FlexItem>
+                <a href={urlBuilder(`content_views/${cv.id}/versions/${versionId}`, '')}>{`Version ${version}`}</a>
+              </FlexItem>
+            </Flex>
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+      </DescriptionList>
+    );
+  }
+
+  return (
+    <GridItem xl2={3} md={6} lg={5}>
+      <Card isHoverable>
+        <CardHeader>
+          <CardTitle>{__('Content View Details')}</CardTitle>
+        </CardHeader>
+        <CardBody>
+          {contentViewDetails}
```

Comment:
A (kinda strong) personal preference here: I don't like when JSX is outside the `return` or render method.  Instead of defining a const `contentViewDetails` above, I'd prefer using conditional rendering here, something like

```jsx
{contentViewDetails.content_facet_attributes &&
<DescriptionList isAutoColumnWidths>
// all the other JSX
}
```

This does raise the question though, if there is no data, should we even be showing the card?